Carmunication/Controllers/MyPage/FriendListViewController.swift

```diff
@@ -66,6 +67,21 @@ final class FriendListViewController: UIViewController {
         navigationItem.backButtonTitle = ""
         navigationController?.navigationBar.tintColor = UIColor.semantic.accPrimary
     }
+
+    // 친구 목록에 값이 있는지 확인하고, 뷰에 표시해줄 요소를 결정하는 메서드
+    private func checkFriendListIsEmpty(friendListCount: Int) {
+        if friendListCount < 1 {
+            // 친구 목록에 친구가 있으면 테이블 뷰를 보여준다.
+            friendListView.friendListTableView.isHidden = true
+            friendListView.emptyView.isHidden = false
+            friendListView.emptyFriendLabel.isHidden = false
+        } else {
+            // 친구 목록에 친구가 없으면 테이블 뷰를 숨기고 친구가 없을 때의 화면을 보여준다.
+            self.friendListView.friendListTableView.isHidden = false
+            self.friendListView.emptyView.isHidden = true
+            self.friendListView.emptyFriendLabel.isHidden = true
```

Comment:
else문에 있는 부분도 self. 삭제하면 좋을 것 같아요

```suggestion
            friendListView.friendListTableView.isHidden = false
            friendListView.emptyView.isHidden = true
            friendListView.emptyFriendLabel.isHidden = true
```